iam/src/index.ts

```diff
@@ -397,7 +397,7 @@ app.post("/api/v0.0.0/verify", (req: Request, res: Response): void => {
       if (verified && currentIssuer === challenge.issuer) {
         // pull the address and checksum so that its stored in a predictable format
         const address = utils.getAddress(
-          utils.verifyMessage(challenge.credentialSubject.challenge, payload.proofs.signature)
+          await verifyDidChallenge(signedChallenge, challenge.credentialSubject.challenge)
```

Comment:
The grants stack integration is using the old challenge format.
We could ask them to update, it wouldn't change the UX much. But it may not be trivial to change.

We could also see if the signedChallenge was passed in, and if not fall back to the old method?

Should we do something like:
signedChallenge ? verifyDidChallenge(...) : utils.verifyMessage(...)

Reply:
Yes, I think we should do this.
Or use a different endpoint when using the new signing method.
I can look into this